rpm/maep-qt.spec

```diff
@@ -69,6 +69,7 @@ install -m 644 -p /usr/lib/libfreetype.so.6 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicui18n.so.52 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicuuc.so.52 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicudata.so.52 %{buildroot}/usr/share/%{name}/lib/
+install -m 644 -p /lib/ld-linux.so.? %{buildroot}/usr/share/%{name}/lib/
 
```

Comment:
Sorry, I don't see the point of these two additions. The package built with the SDK I have (seems to be latest, when I click on update there is no new version taht is proposed) complies with ```rpmvalidator.sh```. Maybe I missed something ?